File: ThalesTest/Models/Employees.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ThalesTest.Models
{
    public class Employees : EmployeeBase
    {
        public List<EmployeeData> data { get; set; }
    }

    public class Employee : EmployeeBase
    {
        public EmployeeData data { get; set; }
    }

    public class EmployeeData
    {
        public int id { get; set; }

        public string employee_name { get; set; }

        public int employee_salary { get; set; }

        public int employee_age { get; set; }

        public string profile_image { get; set; }

        public int employee_anual_salary { get; set; }
    }
}

File: ThalesTest/Scripts/Employees.js
function searchEmployees() {
    var idEmployee = $("#txtEmployeeId").val();

    $.ajax({
        url: "/Home/FilterEmployee",
        dataType: "json",
        data: { idEmployee: idEmployee },
        type: "GET",
        success: function (arr) {
            console.log('employees', arr);
            $("#tbody-employees").empty();
            if (arr.status === "success") {
                if (arr.data === null) {
                    alert("That employee doesn't exist in our database");
                } else {
                    for (let employee of arr.data) {
                        $("#tbody-employees").append(
                            '<tr><td>' + employee.employee_name +
                            '</td><td>' + employee.employee_salary +
                            '</td><td>' + employee.employee_age +
                            '</td><td>' + employee.employee_anual_salary +
                            '</td><td>' + employee.profile_image +
                            '</td></tr>');
                    }
                }
            } else {
                alert(arr.message);
            }
        },
        error: function (data) {

        }
    });
}